Java/AutoML/automl-cms-api/src/main/resources/application-dev.yml

```diff
@@ -0,0 +1,23 @@
+server:
+    port: 8081   # web\u5E94\u7528\u670D\u52A1\u7AEF\u53E3
+
```

Comment:
Maybe we need to set the same globalPrefix as the test configuration.

Reply:
ok， In addition to prod， You mean  we need dev, test.. environment, globalPrefix is  /v1beta1

Reply:
so,  prod use globalPrefix is /v1,   others environment use globalPrefix is /v1beta1  ?  
<img width="660" alt="image" src="https://user-images.githubusercontent.com/48853704/185658726-ab982f80-90c7-42bc-9620-d8f78dcae253.png">

Reply:
> so, prod use globalPrefix is /v1, others environment use globalPrefix is /v1beta1 ? <img alt="image" width="660" src="https://user-images.githubusercontent.com/48853704/185658726-ab982f80-90c7-42bc-9620-d8f78dcae253.png">

Actually all environments will use /v1beta1, we will enable v1 only when our platform and APIs are stable. The stability doesn't depend on the environment.

Reply:
ok， i will update all environment globalPrefix is /v1beta1